crates/re_types/definitions/rerun/blueprint/components/auto_space_views.fbs

```diff
@@ -9,9 +9,7 @@ namespace rerun.blueprint.components;
 
 // ---
 
-/// Whether or not space views should be created automatically.
-///
-/// Unstable. Used for the ongoing blueprint experimentations.
+/// Whether or not Space Views should be created automatically.
```

Comment:
```suggestion
/// Whether or not space views should be created automatically.
```

https://github.com/rerun-io/rerun/blob/main/DESIGN.md#casing

Reply:
ah, should have known